src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.html

```diff
@@ -1,2 +1,3 @@
+<button (click)="editLayoutGrid.addBox()">Add Box</button>
 <button (click)="saveLayout()">Save</button>
-<dot-edit-layout-grid #editLayoutGrid [pageView]="pageView | async"></dot-edit-layout-grid>
+<dot-edit-layout-grid #editLayoutGrid [pageView]="pageView"></dot-edit-layout-grid>
```

Comment:
I know I said we should use ```| async``` but after seen the whole picture I realize it's not necessary.